iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings

```diff
@@ -1969,15 +1969,15 @@
 /* These translations are used for all type.highway.*.bridge. */
 "type.highway.footway.bridge" = "Puente";
 
-"type.highway.footway.demanding_alpine_hiking" = "Camino";
+"type.highway.footway.demanding_alpine_hiking" = "Path";
```

Comment:
ditto here and below

Reply:
ah this might be my fault, i haven't checked out icu/icu. will fix 👍